dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UiPluginServiceImpl.java

```diff
@@ -45,6 +53,14 @@ public class UiPluginServiceImpl extends BaseServiceImpl implements UiPluginServ
     @Autowired
     PluginDefineMapper pluginDefineMapper;
 
+    @Autowired
+    DsVersionMapper dsVersionMapper;
+
+    @Autowired
+    private DsVersionDao dsVersionDao;
```

Comment:
It's better to use dao in service but not mapper.
```suggestion
    @Autowired
    private DsVersionDao dsVersionDao;
```

Reply:
> It's better to use dao in service but not mapper.

You are right. The actual method used is dsVersionDao, which was not deleted during the last modification. I will delete it immediately
![image](https://github.com/apache/dolphinscheduler/assets/45991723/8344ade1-796d-4358-83f0-6e0ea6015a18)